robocop/checkers/naming.py

```diff
@@ -679,7 +703,11 @@ def visit_KeywordCall(self, node):  # noqa
             if len(node.data_tokens) < 2:
                 return
             token = node.data_tokens[1]
-            if token.type == Token.ARGUMENT and not token.value.isupper():
+            if token.type != Token.ARGUMENT or not token.value:
```

Comment:
Second data token of set variable keyword will be always token.Argument so this condition is always false

Reply:
Please take a look now.